diesel_cli/Cargo.toml

```diff
@@ -46,10 +46,14 @@ url = { version = "2.1.0" }
 [features]
 default = ["postgres", "sqlite", "mysql"]
 postgres = ["diesel/postgres", "url", "uses_information_schema"]
-sqlite = ["diesel/sqlite"]
 mysql = ["diesel/mysql", "url", "uses_information_schema"]
-barrel-migrations = ["migrations_internals/barrel", "barrel"]
+sqlite = ["diesel/sqlite"]
 sqlite-bundled = ["sqlite", "libsqlite3-sys/bundled"]
+barrel-migrations = ["migrations_internals/barrel", "barrel"]
+barrel-postgres = ["postgres", "barrel-migrations", "barrel/pg"]
+barrel-mysql = ["mysql", "barrel-migrations", "barrel/mysql"]
+barrel-sqlite = ["sqlite", "barrel-migrations", "barrel/sqlite3"]
+barrel-sqlite-bundled = ["sqlite-bundled", "barrel-migrations", "barrel/sqlite3"]
```

Comment:
There is really no way to prevent duplicating all those features here? I do not really like this.

Reply:
We don't actually need them here. I added them to make it easy for people install diesel_cli. Instead of running,

```
cargo install diesel_cli --no-default-features --features "postgres, barrel-migrations, barrel/pg"
```

They can now just run:

```
cargo install diesel_cli --no-default-features --features barrel-postgres
```

Since they are just mapping to other features, they don't cost us anything.

Reply:
Just going through all the old PR, so to continue here I would like to see this features to be removed. I think it's fine to just pass the 3 feature flags as shown above.